classes/Employee.php

```diff
@@ -584,6 +584,13 @@ public function isSuperAdmin()
      */
     public function getImage()
     {
+        $imagePath = $this->image_dir . $this->id . '.jpg';
+        if (file_exists($imagePath) && is_file($imagePath)) {
```

Comment:
```suggestion
        if (file_exists($imagePath)) {
```
You try to reach a jpg file, not sure it's needed

Reply:
Approved ;)